modules/react/icon/lib/SystemIcon.tsx

```diff
@@ -1,52 +1,53 @@
 import * as React from 'react';
-import {iconColors} from '@workday/canvas-kit-react/tokens';
+import {CanvasColor, iconColors} from '@workday/canvas-kit-react/tokens';
 import {CanvasSystemIcon, CanvasIconTypes} from '@workday/design-assets-types';
 import {CSSObject} from '@emotion/styled';
 import {Icon, IconProps} from './Icon';
 import {createComponent} from '@workday/canvas-kit-react/common';
+import {getColor} from './utils';
 
 export interface SystemIconStyles {
   /**
    * The accent color of the SystemIcon. This overrides `color`.
    */
-  accent?: string;
+  accent?: CanvasColor | (string & {});
```

Comment:
We have this type in other places. The `SystemPropValues` comes from `@workday/canvas-kit-react/layout`
```suggestion
  accent?: SystemPropValues['color'];
```

Same with all the color prop values

Reply:
Addressed.